atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/asts/progress/DefaultBucketProgressStore.java

```diff
@@ -51,7 +51,7 @@ final class DefaultBucketProgressStore implements BucketProgressStore {
 
     public static BucketProgressStore create(KeyValueService keyValueService) {
         return new DefaultBucketProgressStore(
-                keyValueService, BucketProgressPersister.create(ObjectMappers.newServerSmileMapper()));
+                keyValueService, BucketProgressPersister.create(ConsistentOrderingObjectMapper.OBJECT_MAPPER));
```

Comment:
See concerns for a potential refactor. Depending on what we decide, I'll adjust the tests here to use the new object mapper, especially for the "golden" serde tests.

Reply:
Yeah I think the refactor you propose is a good one (all of these classes should use `ObjectPersister`, unless this turns out to be hard / there's some concern I'm not aware of).

Reply:
This will be a separate PR, to avoid bloating this one and causing chain headaches